### PR TITLE
Vpnaas: Update IPSec Policy

### PR DESCRIPTION
**Pull request by simonre**
For #723 

Links to the line numbers/files in the OpenStack source code that support the
code in this PR:

https://github.com/openstack/neutron-vpnaas/blob/058469e1b99b647537a5228c6a384d93df5484df/neutron_vpnaas/db/vpn/vpn_db.py#L420

### CODE REVIEW

**Comment by simonre** [comment]
@jtopjian This is ready for review

File: openstack/networking/v2/extensions/vpnaas/ipsecpolicies/testing/requests_test.go [hunk 1]
```diff
@@ -166,3 +166,80 @@ func TestDelete(t *testing.T) {
...
+		th.TestJSONRequest(t, r, `
+	{
+		"ipsecpolicy":{
+	"name": "updatedname",
+	"description": "updated policy",
+	"lifetime": {
+	"value": 7000
+	}
+	}
+	}
```
Comment (jtopjian) [inline openstack/networking/v2/extensions/vpnaas/ipsecpolicies/testing/requests_test.go]: It looks like the formatting here has gone awry. Normally some mismatched indentation is OK but let's fix this block up.

File: acceptance/openstack/networking/v2/extensions/vpnaas/ipsecpolicy_test.go [hunk 2]
```diff
@@ -21,7 +21,17 @@ func TestIPSecPolicyCRUD(t *testing.T) {
...
+	tools.PrintResource(t, policy)
+
+	updatedDescription := "Updated policy description"
+	updateOpts := ipsecpolicies.UpdateOpts{
+		Description: &updatedDescription,
+	}
 
+	policy, err = ipsecpolicies.Update(client, policy.ID, updateOpts).Extract()
+	if err != nil {
+		t.Fatalf("Unable to update firewall: %v", err)
```
Comment (jtopjian) [inline acceptance/openstack/networking/v2/extensions/vpnaas/ipsecpolicy_test.go]: `s/firewall/policy/`

**Comment by simonre** [comment]
@jtopjian This is ready. This can be merged, it does not depend on anything else.

**Comment by jtopjian** [comment]
LGTM!

**Pull request merged**